docker/acdcserver/install.sh

```diff
@@ -72,5 +72,6 @@ sed -i -e "s#ServerMonitor/2.0#ServerMonitor-acdcserver#g" /data/srv/current/con
 # add proxy generation via robot certificate
 # disable workqueue/reqmon/couch on acdcserver pod
 crontab -l | egrep -v "workqueue|reqmon|couchdb|reboot" > /tmp/mycron
```

Comment:
here you left out reboot, but define it in a line below, please clean-up accordingly.